executor/runtime/container.go

```diff
@@ -45,6 +49,8 @@ func NewContainer(taskID string, titusInfo *titus.ContainerInfo, resources *runt
 
 	env := cfg.GetNetflixEnvForTask(titusInfo, strMem, strCPU, strDisk, strNetwork)
 	labels[appNameLabelKey] = titusInfo.GetAppName()
+	labels[ownerEmailLabelKey] = titusInfo.PassthroughAttributes[ownerPassThroughKey]
+	labels[jobTypeLabelKey] = titusInfo.PassthroughAttributes[jobTypePassThroughKey]
```

Comment:
It's my understanding that if a value isn't present in the map you get the `zero value` which for strings is the empty string, which for me is desired behavior here.